src/main/java/com/mraof/minestuck/skaianet/Session.java

```diff
@@ -27,6 +28,8 @@
 	
 	final Map<PlayerIdentifier, PredefineData> predefinedPlayers;
 	final Set<SburbConnection> connections;
+	final GristGutter gutter;
```

Comment:
How the grist gutter is handled when sessions get merged or split is a pretty important topic that is not yet addressed.